doc/install/plugin.rst

```diff
@@ -33,4 +33,4 @@ Windows install
 A plugin for Windows is currently being developed; it will be posted to the
 DAGMC plugins page when it is ready.
 
-..  _DAGMC_plugins: https://go.wisc.edu/svalinn-cubit-plugin
+..  _DAGMC_plugins: https://github.com/svalinn/Cubit-plugin/releases/tag/0.1.0
```

Comment:
Maybe we should just reference the release page `https://github.com/svalinn/Cubit-plugin/releases` without a specific release, which will allow this to be generic across future releases.